lib/workers/repository/init/inherited.ts

```diff
@@ -99,5 +101,35 @@ export async function mergeInheritedConfig(
       'Removed global config from inherited config.',
     );
   }
-  return mergeChildConfig(config, filteredConfig);
+
+  const decryptedConfig = await decryptConfig(
+    filteredConfig,
+    config.repository,
+  );
+  let returnConfig = decryptedConfig;
+  if (is.nonEmptyArray(returnConfig.extends)) {
```

Comment:
```suggestion
  if (is.nullOrUndefined(returnConfig.extends)) {
     return mergeChildConfig(config, decryptedConfig);
  }
```
Invert the if clause, to make it more easy to readable